context/context.go

```diff
@@ -37,6 +37,7 @@ type Context struct {
 	Git          GitInfo
 	Binaries     map[string]map[string][]Binary
 	Artifacts    []string
+	Checksums    []string
```

Comment:
why is this needed?

Reply:
So that you can only sign the checksum file